src/Command/RecipesCommand.php

```diff
@@ -0,0 +1,109 @@
+use Symfony\Flex\Lock;
+use Symfony\Flex\Recipe;
+
+class RecipesCommand extends BaseCommand
+{
+    /** @var \Symfony\Flex\Flex */
+    private $flex;
+
+    public function __construct(/* cannot be type-hinted */ $flex)
+    {
+        $this->flex = $flex;
+
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('symfony:recipes:show')
+            ->setAliases(['symfony:recipes'])
+            ->setDescription('Shows information about all available recipes.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $locker = $this->getComposer()->getLocker();
+        $lockData = $locker->getLockData();
+
+        // Merge all packages installed
+        $packages = array_merge($lockData['packages'], $lockData['packages-dev']);
+
+        $installedRepo = $this->getComposer()->getRepositoryManager()->getLocalRepository();
+
+        $operations = [];
+        foreach ($packages as $key => $value) {
+            if (null === $pkg = $installedRepo->findPackage($value['name'], '*')) {
+                $this->getIO()->writeError(sprintf('<error>Package %s is not installed</error>', $value['name']));
+
+                continue;
+            }
+
+            $operations[] = new InformationOperation($pkg);
+        }
+
+        $recipes = $this->flex->fetchRecipes($operations);
+        ksort($recipes);
+
+        if (\count($recipes) <= 0) {
+            $this->getIO()->writeError('<error>No recipe found</error>');
+
+            return 1;
+        }
+
+        $write = [
+            '',
+            '<bg=blue;fg=white>                      </>',
+            '<bg=blue;fg=white> Recipes available.   </>',
+            '<bg=blue;fg=white>                      </>',
+            '',
+        ];
+
+        $symfonyLock = new Lock(getenv('SYMFONY_LOCKFILE') ?: str_replace('composer.json', 'symfony.lock', Factory::getComposerFile()));
+
+        /** @var Recipe $recipe */
+        foreach ($recipes as $name => $recipe) {
+            $lockRef = $symfonyLock->get($name)['recipe']['ref'] ?? null;
+
+            $additional = '';
+            if ($recipe->isAuto()) {
+                $additional = '<comment>(auto recipe)</comment>';
+            } elseif (null === $lockRef && null !== $recipe->getRef()) {
+                $additional = '<comment>(new recipe available)</comment>';
+            } elseif ($recipe->getRef() !== $lockRef) {
+                $additional = '<comment>(update available)</comment>';
+            }
+            $write[] = sprintf(' * %s %s', $name, $additional);
+        }
+
+        // TODO : Uncomment the lines following the implemented features
+        //$write[] = '';
+        //$write[] = '<fg=blue>Run</>:';
+        //$write[] = ' * composer symfony:recipes vendor/package to see details about a recipe.';
+        //$write[] = ' * composer symfony:recipes:update vendor/package to update that recipe.';
```

Comment:
I think to make this new command "complete", we would need to at least be able to show 2 commands down here:

A) A command to "update" a single recipe - so `symfony:recipe:update` - item (3) from #518 
B) A command to "install" missing recipes - so `symfony:recipe:install` - item (2) from #518 

The `symfony:recipes vendor/package` or whatever we call it - item (4) from #518 would be super nice to have, as would `symfony:recipes:blame` - but those could come later.

Reply:
- I started the `symfony:recipes vendor/package` in this command. I will update this PR when it's ready
- The `symfony:recipes:install` is ready here : #565 
- For the `symfony:recipes:update`, it would be better to have some feedback on the #565 before I start it
- And for the `symfony:recipes:blame`, it will be a lot more complicated and I agree to do it later